packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -680,9 +678,7 @@ export const init = (force = false) => async (dispatch: Dispatch, getState: GetS
     return true;
 };
 
-export const setEditing = (payload: string | undefined) => (dispatch: Dispatch) => {
-    dispatch({
-        type: METADATA.SET_EDITING,
-        payload,
-    });
-};
+export const setEditing = (payload: string | undefined): MetadataAction => ({
```

Comment:
payload?: string

Reply:
depends on taste. If there is a single method that "adds" "removes" something depending on whether argument is provided or not, I personally prefer to type it as required param with union `something | undefined`

Reply:
good :) it's your playground, i'll leave it as is